spec/rails_helper.rb

```diff
@@ -43,9 +45,10 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  # config.use_transactional_fixtures = false
```

Comment:
Could you add a comment as to the errors that caused you comment out this line. I know we talked about it, but it could be useful for other developers in the future.

Reply:
Ok I will